framework/src/main/java/org/checkerframework/framework/type/TypeFromTypeTreeVisitor.java

```diff
@@ -243,68 +244,56 @@ public AnnotatedTypeMirror visitWildcard(WildcardTree node, AnnotatedTypeFactory
     }
 
     /**
-     * Returns an AnnotatedTypeMirror for uses of type variables with annotations written explicitly
-     * on the type parameter declaration and/or its upper bound.
+     * If a tree is can be found for the declaration of the type variable {@code type}, then a
+     * {@link AnnotatedTypeVariable} is returned with explicit annotations from the type variables
+     * declared bounds. If a tree cannot be found, then {@code type} is returned.
      *
-     * <p>Note for type variable uses in method signatures, explicit annotations on the declaration
-     * are added by {@link TypeFromMemberVisitor#typeVarAnnotator}.
+     * @param type type variable used to find declaration tree
+     * @param f annotated type factory
+     * @return the AnnotatedTypeVariable from the declaration of {@code type} or {@code type} if no
```

Comment:
Inlining the call to `asUse()` sounds like a good simplification, but it would be good to document that change in behavior. At the moment it sounds like `type` is returned directly.
Maybe also the method name `getTypeVariable*Use*FromDeclaration` could highlight that.